src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCount.java

```diff
@@ -14,6 +26,13 @@
     private final SimpleInterval interval;
     private final int refReadCount, altReadCount;
 
+    // constants for Brent optimization of minor allele fraction
+    private static final MaxEval BRENT_MAX_EVAL = new MaxEval(100);
+    private static final double MINOR_ALLELE_FRACTION_RELATIVE_TOLERANCE = 0.01;
```

Comment:
It would cost some extra time but probably quite little, especially with caching.  I'll add an extra order of magnitude.